src/components/AppShelf/AppShelf.tsx

```diff
@@ -94,17 +95,12 @@ const AppShelf = (props: any) => {
         onDragEnd={handleDragEnd}
       >
         <SortableContext items={config.services}>
-          <Grid gutter="xl" align="center">
+          <Grid gutter="lg" align="center">
```

Comment:
Was this change intentional or by accident?

Reply:
It was intentional